packages/nextjs/app/token-vendor/page.tsx

```diff
@@ -117,25 +117,29 @@ const TokenVendor: NextPage = () => {
               <span className="font-bold ml-1">
                 {parseFloat(formatEther(yourTokenBalance?.toString() || 0n))}
               </span>
+              <span className="font-bold ml-1">GLD</span>
```

Comment:
Next time if we hardcode values, we must explicitly leave a comment, like `FixMe: Remove GLD hardcode value! `

Reply:
Change your PR as draft PR since you know is not ready yet

Reply:
fixed